pkgs/servers/home-assistant/default.nix

```diff
@@ -0,0 +1,55 @@
+#{ stdenv, buildPythonApplication, fetchPypi, callPackage
+#, requests, pyyaml, pytz, pip, jinja2, voluptuous, typing, aiohttp, yarl, async-timeout, chardet, astral, certifi
+#, sqlalchemy, netdisco, xmltodict, user-agents
+{ stdenv, python3
+, extraPackages ? x: [] }:
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      yarl = super.yarl.overridePythonAttrs (oldAttrs: rec {
+        name = "${oldAttrs.pname}-${version}";
+        version = "0.18.0";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "11j8symkxh0ngvpddqpj85qmk6p70p20jca3alxc181gk3vx785s";
+        };
+      });
+      aiohttp = super.aiohttp.overridePythonAttrs (oldAttrs: rec {
+        name = "${oldAttrs.pname}-${version}";
```

Comment:
same here

Reply:
`aiohttp` still had a name attribute. I've removed it now.